corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -528,7 +531,11 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             e.preventDefault();
 
             self.validateAllFields().done(function () {
-                FormplayerFrontend.trigger("menu:query", self.getAnswers(), self.selectValuesByKeys);
+                FormplayerFrontend.trigger(
+                    "menu:query",
+                    self.getAnswers(),
+                    self.selectValuesByKeys,
+                    self.forceManualSearch);
```

Comment:
will this always be `false` if `this.options.sidebarEnabled` is `false` ? Just want to check this won't break the existing Search Again workflow if SSCS is not enabled.

Reply:
Yes, that's correct. This should have zero visible effect when not using split screen case search (though `forceManualSearch` will now be explicitly `false` by default).

Reply:
I tested this on staging and it seems to break the Search Again workflow in non SSCS mode. When user clicks Search Again and search, `forceManualSearch` is set to `false` causing the workflow to break. The other weird thing is Web Apps is sending 2 requests one with `forceManualSearch` set to `true` and another set to `false` which sounds like another bug.

Reply:
You're right that this shouldn't default to `false`. I think the property is being implicitly set elsewhere for some workflows, which I don't love. But rather than deal with that logic in this PR I'm opting to only set the property if it needs to be true, otherwise we won't touch it. https://github.com/dimagi/commcare-hq/pull/33199/commits/7c3c7ce59e60b7e5f07aa68ec8eadc35662b5d57

Reply:
Thanks, Seems to be working as expected now.